### PR TITLE
DEPLOY-774: Refactor persistence on master

### PR DESCRIPTION
**Pull request by sergiuv2020**

### CODE REVIEW

File: helm/alfresco-infrastructure/values.yaml [hunk 1]
```diff
@@ -30,7 +20,7 @@ nginx-ingress:
 alfresco-infrastructure:
   alfresco-identity-service:
     enabled: true
-  alfresco-event-gateway:
```
Comment (rgauss) [inline helm/alfresco-infrastructure/values.yaml:33]: Why are we changing this for a persistence PR?

Comment (sergiuv2020) [reply helm/alfresco-infrastructure/values.yaml:33]: fixed, was a typo.

**Review by rgauss — APPROVED** [review]

**Review by cristina-sirbu — APPROVED** [review]

File: helm/alfresco-infrastructure/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-description: A Helm chart Providing the required Alfresco infrastructure for the Digital Business Platform. This includes activemq, alfresco-identity-service, akfresco-event-gateway nginx-ingress and a persistent volume to use with the DBP chart.
+description: A Helm chart Providing the required Alfresco infrastructure for the Digital Business Platform. This includes activemq, alfresco-identity-service, nginx-ingress and a persistent volume to use with the DBP chart.
 name: alfresco-infrastructure
-version: 5.1.1
+version: 5.1.1-DEPLOY-774
```
Comment (jamalkm) [inline helm/alfresco-infrastructure/Chart.yaml]: Is the version right?

Comment (sergiuv2020) [reply helm/alfresco-infrastructure/Chart.yaml]: Changing this now since i have the needed reviews.

**Pull request merged**